src/libs/customError.js

```diff
@@ -0,0 +1,6 @@
+export class RaceError extends Error {
+  constructor(message) {
+    super("[ERROR] " + message);
```

Comment:
```suggestion
    super(`[ERROR] ${message}`);
```
는 어떠실까요?? airbnb js 스타일 가이드 [레퍼런스](https://github.com/airbnb/javascript?tab=readme-ov-file#es6-template-literals)를 참고해봤습니다!

Reply:
오우, 이런 기본적인 것을 놓치고 있었네요. 이런 점의 코드리뷰의 순기능 같습니다. 알려주셔서 무척 감사드립니다!! 🙇‍♂️🙇‍♂️